src/components/formConstants.js

```diff
@@ -0,0 +1,23 @@
+
+const ageGroups = ["18-49", "50-59", "60-79", "80+"]
```

Comment:
just an FYI: I'll change the age groups here and break 18-49 into 18-29, 29-39, 40-49 to make it more flexible
I'll do that in my PR so you don't need to worry about it